doc/rados/api/python.rst

```diff
@@ -319,6 +319,13 @@ exists,  and delete a pool.
 .. automethod:: Rados.delete_pool(pool_name)
 
 
+CLI Commands
+------------
+
+This is a way to run Ceph CLI commands via the librados Python API.
```

Comment:
this is not accurate. as we know, "ceph" cli is a front-end of ceph cluster. it allows user to access different facilities by talking to monitor, mgr and osd.

monitor serves as a yellow book for the cluster. and it is listed in the "yellow book" as well. monitor collects all the available commands served by itself, mgr and osd. that's why ceph cli is able to tell if a command is valid, and to perform basic sanity check. but ceph cli does not always connect to monitor for executing commands. it will try to connect to mgr if it's a mgr command, and talk to osd if it's a osd command on demand.

Reply:
so in addition to `Rados.mon_command()`, we have `Rados.mgr_command`, `Rados.osd_command()` and even `Rados.pg_command()`.

Reply:
> monitor serves as a yellow book for the cluster. and it is listed in the "yellow book" as well. monitor collects all the available commands served by itself, mgr and osd. that's why ceph cli is able to tell if a command is valid, and to perform basic sanity check. but ceph cli does not always connect to monitor for executing commands. it will try to connect to mgr if it's a mgr command, and talk to osd if it's a osd command on demand.

Added a sentence to highlight the fact that one has to choose the correct method and target.

> so in addition to `Rados.mon_command()`, we have `Rados.mgr_command()`, `Rados.osd_command()` and even `Rados.pg_command()`.

Fixed.

Reply:
since you are at this, could fix the `returns` directive of all the `{mgr,osd,pg}_command()`. http://docs.ceph.com/ceph-prs/26934/rados/api/python/#cli-commands.

Reply:
> This is a way to run Ceph CLI commands via the librados Python API

this is kinda weird and even misleading, i'd say.. as ceph cli is actually implemented using these librados python binding. while it puts the other way around.

Reply:
well, to be frank, I just copied some documentation and put it into a PR. I don't plan to continue working on this, till the documentation is perfect. I just want to improve the documentation a tiny bit. 

Will have a look at this PR today evening.

Reply:
> since you are at this, could you fix the `returns` directive of all the `{mgr,osd,pg}_command()`? see http://docs.ceph.com/ceph-prs/26934/rados/api/python/#cli-commands.

fixed

> > This is a way to run Ceph CLI commands via the librados Python API
> 
> this is kinda weird and even misleading, i'd say.. as ceph cli is actually implemented using these librados python binding. while it puts the other way around.

fixed

Also, I added comment to the other parameters in `mon_command`.